library/locale.po

```diff
@@ -80,20 +104,27 @@ msgid ""
 "locale is not changed thereafter, using multithreading should not cause "
 "problems."
 msgstr ""
+"Cela positionne la locale dans toutes les catégories sur le paramètre par "
+"défaut de l'utilisateur (typiquement spécifié dans la variable "
+"d'environnement :envvar:`LANG`). Si la locale n'est pas modifiée par la "
+"suite, l'utilisation du *multithreading* ne devrait pas poser de problèmes."
```

Comment:
Here you can translate multithreading, see 11bc26ab7b77ac12113de773a56f10b611b0d1f6